src/calendar/index.tsx

```diff
@@ -132,12 +136,24 @@ class Calendar extends Component<CalendarProps, CalendarState> {
     enableSwipeMonths: false
   };
 
+  currentDate = this.props.current || this.props.initialDate;
+
   state = {
-    currentMonth: this.props.current ? parseDate(this.props.current) : new XDate()
+    currentMonth: 
```

Comment:
I know the code was already like that, but why are we setting the current date into something called `currentMonth`

Reply:
The calendar only handles the month/year so that's is focus